fdbclient/NativeAPI.actor.cpp

```diff
@@ -4204,7 +4204,7 @@ ACTOR Future< StorageMetrics > extractMetrics( Future<std::pair<Optional<Storage
 	return x.first.get();
 }
 
-ACTOR Future<Standalone<VectorRef<KeyRangeRef>>> getReadHotRanges(Database cx, KeyRange keys) {
+ACTOR Future<Standalone<VectorRef<KeyRangeRef>>> getReadHotRanges(Database cx, KeyRangeRef keys) {
```

Comment:
It seems `getKeyRangeLocations` below will implicitly construct `KeyRange`, since it is expecting a `KeyRange` type. So this optimization is probably not useful.

Reply:
This turns out to be unnecessary change. I reverted this.